--- .tito/lib/origin/tagger/__init__.py
+++ .tito/lib/origin/tagger/__init__.py
@@ -40,5 +40,5 @@ class OriginTagger(VersionTagger):
         super(OriginTagger, self)._tag_release()
 
     def _get_tag_for_version(self, version, release=None):
-        return "v{}".format(version)
+        return "v{}-{}".format(version,release)
 # vim:expandtab:autoindent:tabstop=4:shiftwidth=4:filetype=python:textwidth=0:
